Network section -> Fewer words (#15)
greenelab/biopriors-review: content/04.network-models.md

## Network- and pathway-based models

Rather than operating on sequences, many machine learning models in biomedicine operate on inputs that lack intrinsic order.
Models may make use of gene expression data matrices from RNA sequencing or microarray experiments in which rows represent samples and columns represent genes.

> To take these relationships into account, one might incorporate known relationships or correlations between genes when making predictions or generating a low-dimensional representation of the data.

Reviewer: I think it's slightly unclear what "these relationships" refers to here (probably also true in my original version). 

My attempt at clarifying while keeping a similar word count:

```suggestion
To account for relationships between genes, one might incorporate known interactions or correlations when making predictions or generating a low-dimensional representation of the data.
```
